### PR TITLE
Set up material UI && Integrate with Tailwind CSS

### PR DESCRIPTION
**Pull request by RWambui**
- Tailwind integration
- Material UI integration
- Tailwind injection order
- Breakpoints and global styles config

### CODE REVIEW

File: client/pages/index.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { Button } from "@mui/material";
+import type { NextPage } from "next";
+import Head from "next/head";
+// import Slider from '@mui/material/Slider';
```
Comment (rotimidokun) [inline client/pages/index.tsx:4]: Please remove comment @RWambui

**Review by Maxwell-ihiaso — COMMENTED** [review]
- TailwindCSS implemented correctly 🕶
- MaterialUI implemented correctly 🎏
- Injection order set for Material UI to accept styles from TailwindCSS without conflict 🎈
- General housekeeping is done (Media Queries and global styles) 🎉

File: client/pages/index.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+const Home: NextPage = () => {
```
Comment (rotimidokun) [inline client/pages/index.tsx:6]:
```suggestion
const Home = () => {
```

@RWambui

**Review by rotimidokun — CHANGES_REQUESTED** [review]
Added a few changes to be addressed. @RWambui

**Comment by rotimidokun** [comment]
All changes requested are implemented and PR is good to be merged. @Maxwell-ihiaso comment does a good job at describing what we have implemented @simplytunde

**Comment by simplytunde** [comment]
> All changes requested are implemented and PR is good to be merged. @Maxwell-ihiaso comment does a good job at describing what we have implemented @simplytunde

Give approval once good to go

**Review by rotimidokun — APPROVED** [review]
All changes have been implemented now. Good to merge.

**Pull request merged**